src/main/pages/che-7/administration-guide/examples/che-proc_deploying-kubernetes-image-puller-using-openshift-templates_process.adoc

```diff
@@ -0,0 +1,20 @@
+[subs="+attributes,+quotes"]
+----
+$ for template in $(ls deploy/openshift); \
+  do oc process -f deploy/openshift/$template | oc apply -f -; done
+deployment.apps/kubernetes-image-puller created
+configmap/k8s-image-puller created
+role.rbac.authorization.k8s.io/create-daemonset created
+rolebinding.rbac.authorization.k8s.io/create-daemonset created
+serviceaccount/k8s-image-puller created
+----
++
+[NOTE]
+====
+To set different values than the defaults, use:
+
+[subs="+quotes"]
+----
+$ oc process -p __<parameterName>__=__<value>__ -f deploy/openshift/__<template>__ | oc apply -f -
```

Comment:
I think it would add more clarity to have a note to check the particular template that you are changing the default values for, and get the correct parameter names

Reply:
without a proper example, this instruction could be misleading. Will it be possible to add an example with image overriding?

Reply:
And example ... like https://gist.github.com/tomgeorge/1589e1d4908b63c6ddbae8afbd3c7d00#file-imagepuller-crw-2-1-install ?